src/fides/api/api/v1/endpoints/system.py

```diff
@@ -312,7 +312,6 @@ async def upsert(
 )
 async def delete(
     fides_key: str = Security(
-        verify_oauth_client_for_system_from_fides_key_cli,
```

Comment:
great, I think you found the right spot! Would like to see a test verifying a data steward cannot delete a system

Reply:
You will probably have to swap this though for a different function, probably `verify_oauth_client`.

Reply:
Absolutely. This is what I was mentioning in standup where if we want to create a release branch in an hr, I don't have time as I have an interview now. I'll create a separate ticket though

Reply:
https://ethyca.atlassian.net/browse/PROD-2265

Reply:
This comment needs to be addressed: https://github.com/ethyca/fides/pull/5037#discussion_r1658907279, it needs to be swapped for the right function, not just removed

Reply:
Oh right, and there are more adjustments needed because verify_oauth_client_for_system_from_fides_key_cli returns the fides key but verify_oauth_client does not. I'll look into it.